src/runtime/lpython/lpython.py

```diff
@@ -687,9 +687,9 @@ def get_rtlib_dir():
 
```

Comment:
I'm not sure if the --fast` option works in the C backend. If it does, then we can use it in line 684?

Reply:
It doesn't I think.